app/controllers/items/index.js

```diff
@@ -33,6 +33,18 @@ export default Ember.Controller.extend(SearchMixin, {
     }
   }),
 
+  scannedText: Ember.observer("searchText", function() {
+    const searchInput = this.get("searchText") || "";
+    const sanitizeString = this.sanitizeString(searchInput);
+    //if (sanitizeString) {
```

Comment:
@shreyas098 , why is this commented? If this is not required please remove it.

Keeping a check and then setting the `searchInput` is better.

Please let me know your thoughts.